app/views/advanced_search_finder/show.html.erb

```diff
@@ -26,7 +26,7 @@
 
     <div class="title-and-metadata">
       <%= finder.taxon_link %>
-      <%= render partial: 'govuk_component/title', locals: { title: finder.title } %>
+      <%= render partial: 'govuk_publishing_components/components/title', locals: { title: finder.title } %>
```

Comment:
I don't think we need the `locals:` bit.

Reply:
I think `locals:` is needed when you use the `partial:` syntax. Perhaps in a separate PR we can clean all of those up?